ext/lib/base64/src/base64.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * This file is based on roken from the FreeBSD source.  It has been modified
```

Comment:
This is BSD licensed, not Apache

Reply:
so I should change the commit message right?